airflow/providers/google/cloud/operators/vertex_ai/auto_ml.py

```diff
@@ -607,7 +609,7 @@ class DeleteAutoMLTrainingJobOperator(GoogleCloudBaseOperator):
     AutoMLTabularTrainingJob, AutoMLTextTrainingJob, or AutoMLVideoTrainingJob.
     """
 
-    template_fields = ("training_pipeline", "region", "project_id", "impersonation_chain")
+    template_fields = ("training_pipeline_id", "region", "project_id", "impersonation_chain")
```

Comment:
@eladkal It should be compatible, since `training_pipeline` is just alias to `training_pipeline_id` we do the same things multiple times. But I agree that we need test, for prevent some regression.

@shahar1 You could use `create_task_instance_of_operator` fixture for create task instance and render fields

Sample
https://github.com/apache/airflow/blob/82d9a8961168b0a44d45fe0b6bfc08a8492b6436/tests/providers/apache/livy/operators/test_livy.py#L384-L429

Reply:
Done :)